### PR TITLE
[Docstrings] Doc for Chainer transformer

### PR DESCRIPTION
**Pull request by Fhrozen**
Added some docs for chainer transformer to reduce some error.

### CODE REVIEW

File: espnet/nets/chainer_backend/transformer/label_smoothing_loss.py [hunk 1]
```diff
@@ -21,6 +38,17 @@ def __init__(self, smoothing, n_target_vocab, normalize_length=False):
         self.normalize_length = normalize_length
 
     def forward(self, concat_logit_block, t_block, batch, length):
+        """E2E forward propagation.
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/label_smoothing_loss.py]:
```suggestion
        """Forward E2E.
```

File: espnet/nets/chainer_backend/transformer/layer_norm.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
 # encoding: utf-8
+"""Class Declaration of Transformer's Label Smootion loss."""
 
 import chainer.links as L
 
 
 class LayerNorm(L.LayerNormalization):
+    """Redirects to L.LayerNormalization."""
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/layer_norm.py]:
```suggestion
    """Redirect to L.LayerNormalization."""
```

File: espnet/nets/chainer_backend/transformer/plot.py [hunk 3]
```diff
@@ -1,16 +1,19 @@
 # encoding: utf-8
+"""Class Declaration of Transformer's Attention Plot."""
 
 from espnet.asr import asr_utils
 import logging
 import matplotlib.pyplot as plt
 
 
 def savefig(plot, filename):
+    """Save figure."""
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/plot.py]:
```suggestion
    """Save a figure."""
```

File: espnet/nets/chainer_backend/transformer/plot.py [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
     plot.savefig(filename)
     plt.clf()
 
 
 def _plot_and_save_attention(att_w, filename):
+    """Plot and save attention."""
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/plot.py]:
```suggestion
    """Plot and save an attention."""
```

File: espnet/nets/chainer_backend/transformer/plot.py [hunk 5]
```diff
@@ -64,17 +67,44 @@ def plot_multi_head_attention(data, attn_dict, outdir, suffix="png", savefn=save
 
 
 class PlotAttentionReport(asr_utils.PlotAttentionReport):
+    """Plot attention reporter.
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/plot.py]:
```suggestion
    """Plot an attention reporter.
```

File: espnet/nets/chainer_backend/transformer/plot.py [hunk 6]
```diff
@@ -64,17 +67,44 @@ def plot_multi_head_attention(data, attn_dict, outdir, suffix="png", savefn=save
...
+        device (int | torch.device): Device.
+        reverse (bool): If True, input and output length are reversed.
+        ikey (str): Key to access input (for ASR ikey="input", for MT ikey="output".)
+        iaxis (int): Dimension to access input (for ASR iaxis=0, for MT iaxis=1.)
+        okey (str): Key to access output (for ASR okey="input", MT okay="output".)
+
+    """
+
     def __call__(self, trainer):
+        """Plot and save image file of att_ws matrix."""
```
Comment (sw005320) [inline espnet/nets/chainer_backend/transformer/plot.py]:
```suggestion
        """Plot and save an image file of att_ws matrix."""
```

**Comment by sw005320** [comment]
@Fhrozen, should we wait for this after we merge #1034?

**Comment by Fhrozen** [comment]
yes please, it would be better to wait first for #1034

**Comment by Fhrozen** [comment]
this should target v0.6.0 and not master. 
Can someone change the branch target please.

**Comment by sw005320** [comment]
> this should target v0.6.0 and not master.
> Can someone change the branch target please.

Done. Could you fix the conflicts and also reflect the above suggestions?

**Comment by sw005320** [comment]
LGTM. 
@Fhrozen, ready for merge?

**Comment by Fhrozen** [comment]
Gimme a mins, I will check if it is up-to-date with v.0.6.0 and then just merge please

**Comment by Fhrozen** [comment]
It is already up-to-date. Free to merge pls.

**Comment by sw005320** [comment]
OK, thanks!

**Pull request merged**